includes/APIRead.php

```diff
@@ -83,18 +83,16 @@ function updateDomainManifest($witness_event_id, $db) {
 /**
  * Extension:DataAccounting Standard Rest API
  */
-class StandardRestApi extends SimpleHandler {
+class APIRead extends SimpleHandler {
 
     private const VALID_ACTIONS = [ 
-        'verify_page', 
-        'get_page_by_rev_id', 
-        'page_all_rev', 
-        'get_page_last_rev', 
-        'get_witness_data', 
-        'request_merkle_proof', 
-        'store_signed_tx', 
-        'store_witness_tx', 
-        'request_hash' 
+        'verify_page', //READ
```

Comment:
We already know that the list has all read-only API endpoints. So the `//READ` comments can be removed.

Reply:
Did I delete too much by removing line 97? I need to check that.

Reply:
Done [API_Split e95bf1e] Remove Read comments